docs/index.rst

```diff
@@ -1,11 +1,11 @@
-GeoMesa |release| Manuals
-=========================
+GeoMesa Documentation
+=====================
 
 .. image:: user/_static/img/geomesa-2x.png
 
 .. toctree::
     :maxdepth: 1
 
-    user/index
-    developer/index
-    tutorials/index
+    User Manual <user/index>
+    Developer Guide <developer/index>
+    Tutorials <tutorials/index>
```

Comment:
Yeah, good point - i did this initially then decided to rename the pages themselves.